grails-app/assets/javascripts/spApp/service/loggerService.js

```diff
@@ -37,23 +37,20 @@
                 log: function (category1, category2, data, layerId) {
                     if (paused) return $q.when()
 
-
                     var outputs = []
                     if (layerId) {
                         outputs.push(layerId)
                     }
 
                     history.push({category1: category1, category2: category2, data: data, outputs: outputs})
+                    data["category1"] = encodeURIComponent(category1)
```

Comment:
I'm not sure, do we need to do `encodeURIComponent` here?

Reply:
I think so, it may have some special char, like _, space etc